.github/workflows/frontend_ci.yml

```diff
@@ -0,0 +1,298 @@
+name: Frontend CI
+
+on:
+  pull_request:
```

Comment:
all branches (that match path)?

Reply:
Except for `production`, I've just made an update to fix that.